config/application.rb

```diff
@@ -16,8 +16,6 @@
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
-fields = DfE::Analytics::Fields
-def fields.check!; end
```

Comment:
This was overriding the `check` rake method used by the `dfe:analytics:check` rake task.
As a consequence, the task wasn't running the expected checks when running the tests, causing multiple fields/tables to be missing from `analytics/analytics_blocklist` yaml files.